prometheus/client.go

```diff
@@ -96,6 +96,15 @@ func (in *Client) GetServiceHealth(namespace string, servicename string) Health
 	return getServiceHealth(in.api, namespace, servicename)
 }
 
+// GetNamespaceMetrics returns the Metrics described by the optional service pattern ("" for all), and optional
+// version, for the given namespace. Use GetServiceMetrics if you don't need pattern matching.
+func (in *Client) GetNamespaceMetrics(namespace, servicePattern, version string, duration, step time.Duration,
```

Comment:
so as I understand, for the time being "servicePattern" here is always "" (thus could be removed), but you plan to allow something else for later?

Reply:
@jotak, that's true currently but the logic was basically the same to support any pattern, so I don't think I did very much extra work.  In fact, I wrote this prior to realizing that for now we'd need only to aggregate all services.